host/libhackrf/src/hackrf.h

```diff
@@ -27,31 +27,31 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
 #include <stdint.h>
```

Comment:
Maybe exclude this file from auto-formatting altogether.
It messes with defines and uglifies most function prototypes.

Reply:
Noticed that one can stick a linefeed between blocks of `#define`s to prevent the weird columnizing.

Reply:
The 'weird columnizing' is the `AlignConsecutiveMacros` option. It's something we were doing by hand a lot, so I turned it on, but of course that means it's now applied everywhere.

It has sub-options for whether to align `AcrossEmptyLines` and `AcrossComments`, which are both off by default, so you can always prevent it doing that by separating things.

For the most part I'm happy with it since we tend to group related definitions and separate unrelated ones anyway.

Reply:
Wasn't trying to nitpick – just that the first thing that I happened to see was
```
#define SAMPLES_PER_BLOCK                8192
#define BYTES_PER_BLOCK                  16384
#define MAX_SWEEP_RANGES                 10
#define HACKRF_OPERACAKE_ADDRESS_INVALID 0xFF
#define HACKRF_OPERACAKE_MAX_BOARDS      8
#define HACKRF_OPERACAKE_MAX_DWELL_TIMES 16
#define HACKRF_OPERACAKE_MAX_FREQ_RANGES 8
```
which gave me an aesthetic jolt, but with a linefeed it became
```
#define SAMPLES_PER_BLOCK 8192
#define BYTES_PER_BLOCK   16384
#define MAX_SWEEP_RANGES  10

#define HACKRF_OPERACAKE_ADDRESS_INVALID 0xFF
#define HACKRF_OPERACAKE_MAX_BOARDS      8
#define HACKRF_OPERACAKE_MAX_DWELL_TIMES 16
#define HACKRF_OPERACAKE_MAX_FREQ_RANGES 8
```
which calmed me and instilled hope...
So wanted to inform you about the trick.

Reply:
Yeah, definitely makes sense to separate those two groups. I'll add a commit that splits them up. Let me know if you spot any others.

Reply:
Done.